client/app/components/Input/InputSwitch.scss

```diff
@@ -0,0 +1,38 @@
+@import "../../styles/_global.scss";
+
+.switch {
+  @include setFontSize($size-20);
```

Comment:
Was thinking we can bump this down a tad bit! Was initially thinking 14px but let me know if that seems too tiny.

Was also thinking of bumping the letter spacing by a tad too -- was thinking 0.95em.

Reply:
Haha not sure if 0.95em was a typo but this is what it looks like 
![image](https://user-images.githubusercontent.com/3010728/46581828-8deaa080-c9f3-11e8-8683-0973ca3fb21a.png)

On the desktop, I made the font 16px and on mobile, it will be 14px.
![image](https://user-images.githubusercontent.com/3010728/46581840-b377aa00-c9f3-11e8-95f9-175b1c34ece8.png)

Setting it 14px on desktop was a tad too small for me.